packages/docusaurus-plugin-content-docs/src/sidebars/index.ts

```diff
@@ -91,5 +98,28 @@ export async function loadSidebars(
     sidebarFilePath,
     normalizeSidebarsParams,
   );
-  return processSidebars(normalizedSidebars, options);
+  const categoriesMetadata = Object.fromEntries(
```

Comment:
it's a good practice to separate low-level implementation details to high-level business logic (clean code, single level of abstraction)

I'd rather extract all this in a separate method, that can be tested independently more easily

Reply:
Good point. I was just trying to make it work first :P